### PR TITLE
Change API to header/body 'Color' instead of gray/hue

### PR DESCRIPTION
**Pull request by KyleAMathews**
Also set the default for "headerColor" to 'inherit' so people can easily
override body/header colors on different screens of their site/app.

### CODE REVIEW

**Comment by KyleAMathews** [comment]
@BarryThePenguin

**Comment by KyleAMathews** [comment]
@BarryThePenguin this is a breaking change as yours will be — any guess when you'll be finished? Could make one release for both if you could get a PR in the next few days.

**Comment by KyleAMathews** [comment]
Fixes #92 

Also a nice API simplification — -2 options 👍

File: package.json [hunk 1]
```diff
@@ -10,12 +10,12 @@
     "codecov": "^1.0.1",
     "compass-vertical-rhythm": "^1.3.1",
     "eslint": "^3.5.0",
-    "eslint-config-airbnb": "^11.1.0",
+    "eslint-config-airbnb": "^12.0.0",
     "eslint-plugin-flow-vars": "^0.5.0",
-    "eslint-plugin-import": "^1.15.0",
+    "eslint-plugin-import": "^2.0.1",
```
Comment (BarryThePenguin) [inline package.json]: I think this should stay at `1.16.0` until the airbnb config supports it

If you look at the `package.json` from the [eslint-config-airbnb@12.0.0](https://github.com/airbnb/javascript/blob/eslint-config-airbnb-v12.0.0/packages/eslint-config-airbnb/package.json#L66) release it has a peer dependancy of `eslint-plugin-import@1.16.0`

**Comment by BarryThePenguin** [comment]
I'll have a go at getting a PR together in the next day or two

File: package.json (hunk 1 above)
Comment (KyleAMathews) [inline package.json]: Oh good catch. I'll push a change real quick.

**Comment by BarryThePenguin** [comment]
Looks good 👍

**Pull request merged**